cmd/cluster-init/update_sanitize_prow_jobs.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/dispatcher"
+)
+
+func updateSanitizeProwJobs(o options) error {
+	logrus.Println("Updating sanitize-prow-jobs config")
+	filename := filepath.Join(o.releaseRepo, "core-services", "sanitize-prow-jobs", "_config.yaml")
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+	c := &dispatcher.Config{}
+	if err = yaml.Unmarshal(data, c); err != nil {
+		return err
+	}
+	updateConfig(c, o.clusterName)
+	y, err := yaml.Marshal(c)
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(filename, y, 0644)
+}
+
+func updateConfig(c *dispatcher.Config, clusterName string) {
+	appGroup := c.Groups[api.ClusterAPPCI]
+	jobs := appGroup.Jobs
+	jobs = append(jobs, fmt.Sprintf("pull-ci-openshift-release-master-%s-dry", clusterName))
```

Comment:
I think you want `api.Metadata.JobName()` - make sure to use the `jobconfig.PresubmitPrefix` and associated constants for the prefix

Reply:
How do you find all these!?